### PR TITLE
add epoll_create

### PR DESCRIPTION
**Pull request by ChuanDou2021**

### CODE REVIEW

File: sgx_oc/src/linux/ocall/asyncio.rs [hunk 1]
```diff
@@ -60,6 +60,17 @@ pub unsafe fn epoll_create1(flags: c_int) -> OCallResult<c_int> {
     Ok(result)
 }
 
+pub unsafe fn epoll_create(size: c_int) -> OCallResult<c_int> {
+    let mut result: c_int = 0;
+    let mut error: c_int = 0;
+
+    let status = u_epoll_create_ocall(&mut result as *mut c_int, &mut error as *mut c_int, size);
```
Comment (volcano0dr) [inline sgx_oc/src/linux/ocall/asyncio.rs]: I think `epoll_create` can be implemented by calling `epoll_create1`  in sgx_oc, which reduces defining `EDL` and u_epoll_create_ocall in sgx_urts.

**Comment by volcano0dr** [comment]
@ChuanDou2021 Thanks for your contribution.

File: sgx_oc/src/linux/ocall/asyncio.rs (hunk 1 above)
Comment (ChuanDou2021) [reply sgx_oc/src/linux/ocall/asyncio.rs]: i see, better solution, thanks for your advice

**Comment by volcano0dr** [comment]
LGTM

**Pull request merged**